roles/metrics-export.lua

```diff
@@ -217,6 +235,29 @@ local function validate_http(conf)
     end
 end
 
+local function wrap_handler(handler, metrics)
+    if metrics ~= nil and metrics.enabled == true then
+        local http_middleware = require('metrics.http_middleware')
+        return http_middleware.v1(handler)
+    end
+    return handler
+end
+
+local function routes_equal(old, new)
+    assert(type(old.metrics) == 'table')
+    assert(type(new.metrics) == 'table')
+
+    if old.format ~= new.format then
+        return false
+    end
+
+    if old.metrics.enabled ~= new.metrics.enabled then
+        return false
+    end
```

Comment:
Could it be more compact since we are doing the same thing in both cases?

```suggestion
    if old.format ~= new.format or old.metrics.enabled ~= new.metrics.enabled then
        return false
    end
```

Reply:
Yeah, it makes sense. Thanks!